--- i3bar/src/xcb.c	2014-01-23 16:52:24.000000000 -0500
+++ /home/asarkar/incoming/xcb.c	2014-03-28 23:00:58.287350772 -0400
@@ -1107,7 +1107,7 @@
     font = load_font(fontname, true);
     set_font(&font);
     DLOG("Calculated Font-height: %d\n", font.height);
-    bar_height = font.height + 6;
+    bar_height = font.height + 2;
 
     xcb_flush(xcb_connection);
 
@@ -1716,7 +1716,7 @@
                           outputs_walk->buffer,
                           outputs_walk->bargc,
                           MAX(0, (int16_t)(statusline_width - outputs_walk->rect.w + 4)), 0,
-                          MAX(0, (int16_t)(outputs_walk->rect.w - statusline_width - traypx - 4)), 3,
+                          MAX(0, (int16_t)(outputs_walk->rect.w - statusline_width - traypx - 4)), 1,
                           MIN(outputs_walk->rect.w - traypx - 4, statusline_width), font.height + 2);
         }
 
@@ -1752,7 +1752,7 @@
                               outputs_walk->bargc,
                               mask,
                               vals_border);
-                xcb_rectangle_t rect_border = { i, 1, ws_walk->name_width + 10, font.height + 4 };
+                xcb_rectangle_t rect_border = { i, 1, ws_walk->name_width + 10, font.height + 2 };
                 xcb_poly_fill_rectangle(xcb_connection,
                                         outputs_walk->buffer,
                                         outputs_walk->bargc,
@@ -1771,7 +1771,7 @@
                                         &rect);
                 set_font_colors(outputs_walk->bargc, fg_color, bg_color);
                 draw_text(ws_walk->name, outputs_walk->buffer, outputs_walk->bargc,
-                          i + 5, 3, ws_walk->name_width);
+                          i + 5, 1, ws_walk->name_width);
                 i += 10 + ws_walk->name_width + 1;
 
             }
@@ -1787,7 +1787,7 @@
                           outputs_walk->bargc,
                           mask,
                           vals_border);
-            xcb_rectangle_t rect_border = { i, 1, binding.width + 10, font.height + 4 };
+            xcb_rectangle_t rect_border = { i, 1, binding.width + 10, font.height + 2 };
             xcb_poly_fill_rectangle(xcb_connection,
                                     outputs_walk->buffer,
                                     outputs_walk->bargc,
@@ -1807,7 +1807,7 @@
                                     &rect);
 
             set_font_colors(outputs_walk->bargc, fg_color, bg_color);
-            draw_text(binding.name, outputs_walk->buffer, outputs_walk->bargc, i + 5, 3, binding.width);
+            draw_text(binding.name, outputs_walk->buffer, outputs_walk->bargc, i + 5, 1, binding.width);
 
             unhide = true;
         }
